mobile/lib/firebase_options.dart

```diff
@@ -27,7 +27,13 @@ class DefaultFirebaseOptions {
     }
     switch (defaultTargetPlatform) {
       case TargetPlatform.android:
-        return android;
+        switch (network) {
+          case "regtest":
+            return androidDemo;
+          case "mainnet":
```

Comment:
🤔 maybe also return `androidProd` in this case instead of letting it fall through?